docs/wrappers/ledgerutil_postscript.md

```diff
@@ -0,0 +1,20 @@
+## Example Usage
+
+### ledgerutil compare example
+
+Here is an example of the `ledgerutil compare` command.
+
+  * Compare snapshots from two different peers for mychannel at snapshot height 5.
+
+    ```
+    ledgerutil compare -o ./compare_output -f 10 ./peer0.org1.example.com/snapshots/completed/mychannel/5 ./peer1.org1.example.com/snapshots/completed/mychannel/5
+
+    Both snapshot public state and private state hashes were the same. No results were generated.
+
+    ```
+
+    The response above indicates that the snapshots are identical. If the snapshots were not identical, the command results will indicate where the comparison output files are written.
```

Comment:
Maybe we can also include an example of running the command between two snapshots where there are differences:

'''
ledgerutil compare snapshots/ss1 snapshots/ss2

Successfully compared snapshots. Results saved to mychannel_3_comparison. Total differences found: 3
'''